docs/How-To_Guides/OpenStack/Cinder/encrypted-volumes.md

```diff
@@ -0,0 +1,157 @@
+  --os-volume-api-version 3.66 \
+  -f value \
+  -c encryption_key_id \
+  enc_drive
+```
+
+Instead of the volume name, you can of course also specify its UUID:
+
+```bash
+openstack volume show \
+  --os-volume-api-version 3.66 \
+  -f value \
+  -c encryption_key_id \
+  33211b21-8d4f-48e9-b76f-ec73ffd19def
+```
+
+
+## Deleting an encrypted volume
+
+When you decide you no longer need an encrypted volume and want to
+delete it, you can do so with the `openstack volume delete`
+command. As long as you do this with the same user account as the one
+that created the volume, this will succeed without further
+intervention.
+
+However, if you are trying to delete a volume that was created by a
+different user, you’ll run into the limitation that the *secret*
+associated with the volume is owned by that user. As a result, the
+deletion of the encrypted volume using your own user credentials will
+fail.
+
+There are two options to work around this limitation:
+
+1. You can switch to the user credentials of the user that created the
+   volume (if you have access to them), and proceed with the deletion.
+2. You can ask the user that created the volume to [add you to the
+   Access Control List (ACL) for the
+   secret](../Barbican/share-secret.md). This will enable you to read
+   the secret, and to delete the volume using your own credentials.
+
+
+## Block device encryption caveats
+
+Once a volume is configured to use encryption and is also attached to
+an instance in {{extra.brand}}, some caveats apply that you might want
+to keep in mind.
+
+Sometimes, automatically or through administrator intervention, we
+move one of your instances to another physical machine. This process
+is known as *live migration,* and it normally does not interrupt the
+instance’s functionality at all — typically, neither you nor the
+application users notice that live migration has even happened. This
+is a very common occurrence when we do routine upgrades of the
+{{extra.brand}} platform, during our pre-announced maintenance windows.
+
+The same considerations apply to physical node failure. If the
+physical machine running your instance fails, we can automatically
+recover it onto another machine — an action known as *evacuation.*
+
+Live migration or evacuation *including encrypted volumes* does,
+however, require that whoever does the migration also has at least
+read access to the volume’s encryption secret.
+
+This means that you have two options:
+
+1. If you *do* trust us to include your instances in live migrations
+   and evacuations, even if they attach encrypted volumes, then you
+   can [add our administrative account to the Access Control List
+   (ACL) for your secrets](../Barbican/share-secret.md).
+2. If you *don’t* want to share your secrets but you still want to use
+   encrypted volumes, you should build your own mechanism or process
+   (preferably automated) so that your instances recover in case they
+   become non-functional.
```

Comment:
We should say also that anti-affinity won't be respected here. But we will provide timeframe for manual evacuation. Also I guess this could refference to "Maintenance Window Instructions for instances with encrypted volumes attached" page.

Reply:
That sounds to me like it should be in a separate bit of documentation dealing with live migration and maintenance.